Example/package.json

```diff
@@ -7,6 +7,7 @@
     },
     "dependencies": {
         "react-native": "^0.20.0",
-        "react-native-root-toast": "^0.0.1"
+        "react-native-root-toast": "../",
```

Comment:
Thanks for this PR. :+1: 
But I have one question for this.
I`m not quite understand why should change the dependcy of `react-native-root-toast` to relative path.
Wouldn`t it be better that the example can be installed independently?